src/ssl.c

```diff
@@ -62753,34 +62758,33 @@ WOLFSSL_STACK* wolfSSL_PKCS7_to_stack(PKCS7* pkcs7)
 WOLFSSL_STACK* wolfSSL_PKCS7_get0_signers(PKCS7* pkcs7, WOLFSSL_STACK* certs,
                                           int flags)
```

Comment:
Ok so this function actually looks like a bug in OpenSSL itself. `get0` functions return objects that don't need to be `free`'d while the return of this function does need to be `free`'d. This is exactly what OpenSSL (up to latest master) does so its consistent but misleading.

Reply:
Good observation, I will add a comment note above this function mentioning that the returned stack needs to be freed.